src/Microsoft.AspNet.WebHooks.Custom.SqlStorage/WebHooks/WebHookStoreContext.cs

```diff
@@ -13,15 +16,58 @@ namespace Microsoft.AspNet.WebHooks
     public class WebHookStoreContext : DbContext
     {
         internal const string ConnectionStringName = "MS_SqlStoreConnectionString";
-        private const string ConnectionStringNameParameter = "name=" + ConnectionStringName;
+        private readonly string _tableName;
+        private readonly string _schemaName = "WebHooks";
 
         /// <summary>
         /// Initializes a new instance of the <see cref="WebHookStoreContext"/> class.
         /// </summary>
-        public WebHookStoreContext() : base(ConnectionStringNameParameter)
+        public WebHookStoreContext() : base("name=" + ConnectionStringName)
         {
         }
```

Comment:
Up to you: I suggest setting `NameOrConnectionString` in this constructor overload too.